content/future.html

```diff
@@ -2,104 +2,299 @@ <h1 id="future"> Future </h1>
+    co-op), you know exactly what to pack and where, and it
+    basically requires no time at all to get ready!
+    <br></br>
+    Take the same courses as the people you live with.
+    <br></br>
+    If you don't collect air miles you're missing out on
+    $10-$50 per flight.
+    <br></br>
+    Talk less, listen more.
+    <br></br>
+    Surround yourself with successful people who will naturally
+    push you to do your best.
+    <br></br>
+    Be empathetic.
   </p>
-</p>
+
+  <div class="vb"></div>
+  <span class="big">
+    "What is one piece of advice you would give to your
+    first-year self?"
+  </span>
+  <p class="quote">
+    Have confidence. You deserve to be here.
+    <br></br>
+    Overworking yourself never works! You reach a point
+    were putting more work doesn't yield the cost benefit
+    necessary, instead you will feel much better and
+    perform as well or better by being involved in something
+    you enjoy.
+    <br></br>
+    Don't take everything so seriously. Take the time to
+    learn even if it means lower marks, and take some time
+    to explore projects, hobbies and have fun!
+    <br></br>
+    I don't think my first-year self would have taken my
+    advice.
+    <br></br>
+    Spend more time having fun. Always keep well being and
+    quality of life in mind.
+    <br></br>
+    Don't limit your co-op applications to one place - especially
+    in first year. It's better to apply to a variety of jobs
+    in a variety of locations -- use that as an excuse to
+    lightly travel when on coop.
+    <br></br>
+    Stop sending cringey snaps when drunk.
+    <br></br>
+    Buy doge coin.
+    <br></br>
+    People are less intimidating than you think.
+    <br></br>
+    Take more risks throughout university
+    <br></br>
+    Be yourself.
+  </p>
+
+  <div class="vb"></div>
+  <span class="big">
+    "What was the best lesson you learned in your undergrad?"
+  </span>
+  <p class="quote">
+    Compassion
+    <br></br>
+    How important reflection time is. Taking time to reflect
+    on how I'm feeling about all parts of my life has led to
+    some of the biggest, most positive changes I made during
+    undergrad.
+    <br></br>
+    Failing a class.
+    <br></br>
+    Not all things are worth doing.
+    <br></br>
+    How to learn complex ideas quickly.
+    <br></br>
+    Figure out what you want and don't be afraid to go
+    against the norm. Don't strive for something just
+    because other people do.
+    <br></br>
+    Stress baking helps a lot, surprisingly.
+    <br></br>
+    The importance of being confident in yourself and what you can do.
+    <br></br>
+    Cramming last minute works fantastically, unless it doesn't.
+    <br></br>
+    Mental health is as important as physical health. In tandem,
+    your productivity doesn't define your self worth. It's fine
+    to have a day where you don't get anything significant done.
+    <br></br>
+    Humility
```

Comment:
nit (for the end period)
```suggestion
    Humility.
```